addon/modifiers/did-resize.js

```diff
@@ -38,19 +56,30 @@ export default class DidResizeModifier extends Modifier {
     this.handler = handler;
     this.options = options || this.options;
 
-    this.observe();
+    // Only observe if the modifier is installed
+    if (this._didInstall) {
```

Comment:
Is it possible to check that the `observer` property isn't null, instead of adding this boolean flag?

Reply:
This actually may be an opportunity to tackle #46 if you feel like taking that on. Moving the logic of `didInstall` to a constructor would eliminate the need to do this check at all. We would know that an observer exists because it *has* to be set up by the constructor of the first instance of this modifier, and we wouldn't call `this.observe` in the constructor, waiting instead to call it here within `didReceiveArguments`. What do you think?